_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
+description: Research Page
+tags: [robotics,panda,simulator,research]
+permalink: robotic_skin.html
+---
+
+{% include image.html url="research/skin_unit.png" max-width="100%" description="Flexible skin unit created by etching the circuit diagram into a copper sheet and then encasing it in the flexible polymer ecoflex." %}
+
+
+Robots have been transitioning into human-populated environments and replacing their physical cages used to keep humans safe with complex perception and software. However, current solutions are computationally expensive, prone to occlusion, and require setup overhead. Robots require compact, self-contained sensing of nearby space to provide inherently save interactions. In this work, we present the first prototype of a flexible, conformant whole-body artificial skin for collaborative robotics. In order to locate all skin units on the surface of a robot, a novel kinematically calibration algorithm is required to reduce setup time.
+
+# Contributions
+
+1. <strong>Kinematic Calibration:</strong> We present a framework for automatic kinematic calibration that leverages an Inertial Measurement Unit (IMU) to automatically locate skin units, of arbitrary number, along the surface of a robot.
+
+1. <strong>Skin Unit:</strong> We present a new <strong>flexible artificial skin</strong> that can conform to multiple surfaces. The skin utilizes an IMU and a proximity sensor to sense obstacles at a distance. Future prototypes will embed additional capabilities (e.g. pressure sensing, temperature sensing, tactile sensing...)
```

Comment:
the skin unit contribution should be presented before the kinematic calibration (so that you can also introduce what do you mean with SU)

Reply:
Same with the sections below